website/docs/docs/dbt-cloud-apis/metadata-api.md

```diff
@@ -1,44 +1,85 @@
+- Q2 &mdash; dbt v1.6: Use public models to enable [multi-project deployments](https://github.com/dbt-labs/dbt-core/discussions/6725) and access the global lineage
+- Q2 &mdash; dbt v1.6: New and revised model and entity endpoints to unlock Semantic Layer use cases
+- Q2 &mdash; Pagination for manageable responses and performance when querying long lists of dbt objects
+- Q3 &mdash; Retrieve project information from the dbt logs, such as catalog information, without having to generate documentation.
+
+
+</TabItem>
+
+<TabItem value="quality" label="Quality">
+
+To improve customers’ data quality, we’re enhancing the ability for data teams to configure dbt and monitor for issues in dbt Cloud and integrated tools.  
+
+- Q1 &mdash; Add Metadata API support for dbt Core 1.5 elements of data contracts like model owners, access, and constraints.
+- Q4 or later &mdash; Improvements to webhooks to support notifications about tests, freshness, and events per model rather than per run.
+- Q4 or later &mdash; Streaming results so users can access real-time metadata during a run.
+- Q4 or later &mdash; Integrate with model freshness SLAs for intelligent scheduling.
+
+</TabItem>
+
+<TabItem value="operations" label="Operations">
 
-- **Discovery*** &mdash; Find and understand dbt assets to analyze in integrated tools using information like model and metric definitions, column info, and lineage. One example of this is the [dbt Semantic Layer integration](/guides/dbt-ecosystem/sl-partner-integration-guide). 
-- **Quality*** &mdash; Make sure users have correct and up-to-date data for their analyses by monitoring test failures, source freshness, run status, exposures, and dependencies.
-- **Operations*** &mdash;  Help data teams run dbt efficiently and reduce costs by using historical run data, including information like model build time and run counts.
- 
 
-Users may use the API directly or within an integrated tool, and it powers some experiences in dbt Cloud itself. Many Metadata API use cases also benefit from [webhooks](/docs/deploy/webhooks) to access the most up-to-date project and run information. 
+To improve dbt operations, we’ll enable API users to access and query richer information about execution on dbt Cloud in more ergonomic ways. This is our primary focus in the second half of 2023. 
 
-*_More detailed use case and integration guides coming soon_
+- Q3 &mdash; Enable easier queries across runs to analyze performance over time, such as time aggregation for execution results of individual models and the project as a whole.
+- Q3 &mdash; Provide granular execution information from dbt Cloud, such as invocation history.
+- Q3 &mdash; Improvements to webhooks for notifications about run events.
+- Q4 &mdash; Streaming results to see the DAG as it builds during a run.
```

Comment:
Add "[Later]" at the start of this to make it clear it isn't part of the other, 2023 projects